src/pages/post/index.tsx

```diff
@@ -60,33 +71,50 @@ const PostPage = (): ReactElement => {
   const handleUpdateCategory: SelectOnChangeHandler = ({ code }) => {
     setPostForm({
       ...postForm,
-      category: Number(code)
+      category: code
     })
   }
 
-  // const handleUpdateImageUrls: UploaderOnChangeHandler = ({ images }): void => {
-  //   setPostForm({
-  //     ...postForm,
-  //     imageUrls: images
-  //   })
-  // }
+  const handleUpdateImageInfos: UploaderOnChangeHandler = async ({
+    images: imageInfos
+  }) => {
+    setPostForm(prev => ({
+      ...prev,
+      imageInfos
+    }))
+  }
 
   const handleUpdatePostForm: HandleUpdatePostForm = (e): void => {
-    const { name, value } = e.currentTarget
+    const { name, value } = e.target
 
     setPostForm({
       ...postForm,
       [name]: value
     })
   }
 
-  const handleChangeRadio = (e: ChangeEvent<HTMLFormElement>) => {
-    const { name, value } = e.target
+  const handlePostProduct = async () => {
+    const { imageInfos, price, ...restInfos } = postForm
```

Comment:
```suggestion
    const { imageInfos, price, ...post } = postForm
```
post에 대한 내용이어서 post로 하는 게 더 직관적일 것 같습니다!

Reply:
[변경완료](https://github.com/price-offer/offer-fe/pull/179/commits/17abbec909f742c99514d4e2c6f59e8da13e77e8) !